tools/setup.psm1

```diff
@@ -162,7 +162,14 @@ function Invoke-SHiPSTest {
     if($script:IsWindows){
         $PowerShellExePath = Join-Path -Path $PowerShellHome -ChildPath 'PowerShell.exe'
     } else {
-        $PowerShellExePath = 'powershell'
+        if($script:IsLinux)
```

Comment:
Should the check be for ISCoreCLR? because the exe rename is for PSCore6 on all platform?